src/samples/ExtendedSlashScreen.Uno.Samples/ExtendedSlashScreen.Uno.Samples.Shared/Shell.xaml.cs

```diff
@@ -27,9 +22,13 @@ public Shell(LaunchActivatedEventArgs e)
 
 			Instance = this;
 
-#if WINDOWS_UWP || WINDOWS
+#if WINDOWS_UWP
 			AppExtendedSplashScreen.SplashScreen = e?.SplashScreen;
+#elif WINDOWS
+			AppExtendedSplashScreen.SplashScreen = e?.UWPLaunchActivatedEventArgs.SplashScreen;
```

Comment:
UWP for Windows?

Reply:
That's just the name of the variable, it's from this file:
  public sealed class LaunchActivatedEventArgs 
![image](https://user-images.githubusercontent.com/90481654/202779666-e1a497f2-9f5e-4ad0-9045-472fcc7876fd.png)
Which is clearly a class for WinUI and not UWP. I don't know why it's named that way.

Reply:
Let change the name of the event to avoid confusion

Reply:
But, it's not name I chose, thought? I am not sure what you are asking me to do?

Reply:
This is from microsoft, I think

Reply:
The samples runs with no issues in WinUI? Looking at the doc, the argument should be null.
The definition from [UWPLaunchActivatedEventArgs Property](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.launchactivatedeventargs.uwplaunchactivatedeventargs?view=windows-app-sdk-1.2#microsoft-ui-xaml-launchactivatedeventargs-uwplaunchactivatedeventargs):
Gets the [LaunchActivatedEventArgs](https://learn.microsoft.com/en-us/windows/windows-app-sdk/api/winrt/microsoft.ui.xaml.launchactivatedeventargs?view=windows-app-sdk-1.2) for a UWP app. For desktop applications, this property is null.

Reply:
You're right, I tried it without it and it doesnt seem to change anything, so, I removed it.

Reply:
Oh sorry didn't saw this comment before. Essentially, it's the equivalent of the UWP launch argument class. For compability purposes I guess. Although yeah it's null at the moment because splashscreen is not supported on WinUI https://github.com/microsoft/microsoft-ui-xaml/issues/4055

Reply:
Should I add it back?